scripts/lib/releaseNotes.ts

```diff
@@ -150,3 +151,161 @@ function addNewReleaseNoteToc(releaseNotesNode: any, newVersion: string) {
     });
   }
 }
+
+/**
+ * Sorts the release notes dictionary by the patch version
+ * in descending order, e.g. 0.25.2 then 0.25.1.
+ *
+ * Returns a dictionary with the corresponding markdown for
+ * each patch with the entries sorted by release.
+ */
+export function sortReleaseNotesVersions(markdownByPatchVersion: {
+  [id: string]: string;
+}) {
+  const markdownByPathEntries = Object.entries(markdownByPatchVersion);
```

Comment:
Nit: it's probably clearer to inline this, since you're only using the variable once. I personally find `Object.entries(markdownByPatchVersion).sort()` clearer because I'm thinking in terms of the `Object.entries()` API, rather than having to remember what the variable `markdownByPathEntries` looks like.